Code/C++/bst_traversals.cpp

```diff
@@ -0,0 +1,127 @@
+/* Traversal is a process to visit all the nodes of a tree and may print their values too. Because, all nodes are connected via edges (links) we always start from the root (head) node. That is, we cannot randomly access a node in a tree.
+ There are three ways which we use to traverse a tree −
+
+In-order Traversal:
+
+   1. Traverse the left subtree
+   2. Visit the root.
+   3. Traverse the right subtree
+ Pre-order Traversal:
+ 
+   1. Visit the root.
+   2. Traverse the left subtree
+   3. Traverse the right subtree
+ Post-order Traversal:
+
+   1. Traverse the left subtree
+   2. Traverse the right subtree
+   3. Visit the root.
+*/
+
+#include <iostream> 
+using namespace std; 
+
+struct Node 
```

Comment:
Don't use struct in c++. Use class to implement.

Reply:
ok !! I would change it. Thanks for reviewing the code.